test/lib/rules/ssr/no-unsupported-node-api.js

```diff
@@ -18,6 +18,16 @@ testRule('ssr/no-unsupported-node-api', {
                 }
             `,
         },
+        {
+            code: `
```

Comment:
Do we need more tests or is that newly added test covers the change in the rule?

Reply:
The changes done are only related to skipping code blocks with ssr env check.I think this case covers that.